pkg/kubectl/cmd/get_test.go

```diff
@@ -162,9 +162,9 @@ func TestGetUnknownSchemaObjectListGeneric(t *testing.T) {
 		},
 		"handles second specific version": {
 			output: "unlikelyversion",
-			list:   "v1beta3",
+			list:   "v1",
 			obj1:   "unlikelyversion", // doesn't have v1beta3
-			obj2:   "v1beta3",         // version of the API response
+			obj2:   "v1",              // version of the API response
```

Comment:
Ah, that should have been changed to v1beta3 when we dropped v1beta1 then.  I think v1beta3 was just an additional sanity check.

----- Original Message -----
> >  			obj1:   "unlikelyversion", // doesn't have v1beta3
> > -			obj2:   "v1beta3",         // version of the API response
> > +			obj2:   "v1",              // version of the API response
> 
> Actually, the test factory only supports unlikelyversion and v1beta1:
> ```
> 	scheme.AddKnownTypeWithName("", "Type", &internalType{})
> 	scheme.AddKnownTypeWithName("unlikelyversion", "Type", &externalType{})
> 	scheme.AddKnownTypeWithName("v1beta1", "Type", &ExternalType2{})
> ```
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/9280/files#r31839841